src/internal/storage/server_test.go

```diff
@@ -419,3 +420,28 @@ func checkFilesetCDR(ctx context.Context, t *testing.T, c storage.FilesetClient,
 	}
 	require.Equal(t, 0, len(expected))
 }
+
+func TestRenewFileset(t *testing.T) {
+	ctx := pctx.TestContext(t)
+	env := realenv.NewRealEnv(ctx, t, dockertestenv.NewTestDBConfig(t).PachConfigOption)
+	c := env.PachClient.FilesetClient
+	id, _, err := createFileset(ctx, c, 99, units.KB)
+	require.NoError(t, err)
+	_, err = c.RenewFileset(ctx, &storage.RenewFilesetRequest{
+		FilesetId:  id,
+		TtlSeconds: math.MaxInt64,
+	})
+	require.True(t, strings.Contains(err.Error(), "exceeds max ttl"))
+	_, err = c.RenewFileset(ctx, &storage.RenewFilesetRequest{
+		FilesetId:  id,
+		TtlSeconds: 1800000000000,
```

Comment:
This test case seems unnecessary, and would require maintenance if we changed the max ttl. Also, it looks like there are duplicate max ttl error checks below this.